### PR TITLE
Js functions

### PR DESCRIPTION
**Pull request by Ahmed-Alfaouq**
What I did:
- Added open and close functions for the hamburger menu
- Added scrollspy for menu options

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,321 +1,337 @@
...
-          <li>html</li>
-          <li>bootstrap</li>
-          <li>ruby</li>
+    <header id="navbar">
+      <a href="/" class="logo inter-font">Ahmed Omar</a>
+      <nav id="menu-icons">
+        <span></span>
+        <span></span>
+        <span></span>
+      </nav>
```
Comment (hemant-soni-vst-au4) [inline index.html:51]: - [ ] As per Figma design the logo and hamburger button should be placed on the left and right end respectively on the navbar. Kindly check the design and fix it.

- [ ] The colour of the hamburger button should be white as per Figma design but right now it has some greyish colour. Kindly try to make a design similar to Figma's design as much as possiblele.


Yours 
image

Origianl
image

**Review by hemant-soni-vst-au4 — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far! 👍 
There are some issues that you still need to work on to go to the next project but you are almost there! 

Alt Text
## Required Changes ♻️
_Check the comments under the review._

Cheers and happy coding 😃

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. You can mention me (@hemant-soni-vst-au4) so I can see your comments and respond.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Ahmed-Alfaouq) [reply index.html:51]: @hemant-soni-vst-au4
Kindly, can you check now and tell me if everything is ok?

Comment (hemant-soni-vst-au4) [reply index.html:51]: - Great, Now it looks good.

**Review by hemant-soni-vst-au4 — APPROVED** [review]
Hi Team, 

## Your project is complete!  
There is nothing else to say other than... it's time to merge it :shipit: 

## Congratulations! 🎉 

Cheers and happy coding 😃

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. You can mention me (@hemant-soni-vst-au4) so I can see your comments and respond.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**